### PR TITLE
FEATURE - Improving information about help

### PR DESCRIPTION
**Pull request by brunats**
**- What I did**
Updated the help messages for the core commands and added comments to some public functions found in the modified files.

**- How I did it**

Changing the help messages according to the documentation and adding comments on the necessary public functions (in files already changed previously).

**- How to verify it**

Exec `rit`
Before:
antes

Now:
depois

> Prints updated on Jul 14 at 7:03 pm

**- Description for the changelog**

Improving information about help

### CODE REVIEW

**Comment by brunats** [comment]
> Good job! It became much clearer with examples.
> Do you think it is a good idea not to use the word "objects"? For me it is not valuable information.


Looking again, I agree with you, let's see what the other reviewers think and depending on I remove the 'objects'.

File: pkg/cmd/autocomplete.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ type autocompleteCmd struct {
 func NewAutocompleteCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:     "completion SUBCOMMAND",
-		Short:   "Add autocomplete for terminal",
+		Short:   "Add autocomplete for terminal (bash, zsh)",
```
Comment (marcoscostazup) [inline pkg/cmd/autocomplete.go:25]: We have autocomplete for fish and powershell too now

Comment (brunats) [reply pkg/cmd/autocomplete.go:25]: I am updating to add them through a list.

Comment (brunats) [reply pkg/cmd/autocomplete.go]: Fix in https://github.com/ZupIT/ritchie-cli/pull/256/commits/6c3bf279b21ee10801679832104b3d8aa3364d33.

Comment (marcoscostazup) [reply pkg/cmd/autocomplete.go]: Great! :)

**Pull request merged**